libgit/autogit_node_wrappers.go

```diff
@@ -165,6 +198,25 @@ func (rn *repoNode) finishPopulate(populated bool) {
 	rn.populated = populated
 	close(rn.populatingInProgress)
 	rn.populatingInProgress = nil
+	rn.am.populateDone(rn)
+}
+
+func (rn *repoNode) updated(ctx context.Context) {
+	h, err := rn.am.config.KBFSOps().GetTLFHandle(ctx, rn)
+	if err != nil {
+		rn.am.log.CDebugf(ctx, "Error getting handle: %+v", err)
+		return
+	}
+
+	dstDir := rn.dstDir()
+	rn.am.log.CDebugf(
+		ctx, "Repo %s/%s/%s updated", h.GetCanonicalPath(), dstDir, rn.repoName)
+	_, err = rn.am.Pull(
+		context.Background(), rn.h, rn.repoName, "master", h, dstDir)
```

Comment:
We should make custom contexts for this with tags.

Reply:
Actually, you pass in a `context.Background()` in the one place where it's called, so maybe the tags can come from there, and this `Pull` uses `ctx`.

Reply:
`updated()` is called from a goroutine though, so I didn't want to pass that same `ctx` into a new call (since it might be canceled, etc.).  Maybe it's better to make a new ctx with new tags at the caller and pass it / use it here.  That seems cleaner, I'll do that.

Reply:
(Ah, now I see that's probably what you were suggested with the very next comment.)